### DISCUSSION TITLE
Get users by list of emails

### DISCUSSION THREAD

**Discussion by g-zecchi** [zitadel/zitadel#6828] [Q&A] [answered]
Hello! Is it possible to get multiple users by a list of emails in only one request on /users/_search?
Just like the 'inUserIdsQuery' query is working.

**Comment by hifabienne** [comment, marked as answer]
Hei @g-zecchi 
This is currently not possible.
You can upvote the issue here: https://github.com/zitadel/zitadel/issues/6829
We do not have it yet on the roadmap

**Reply by g-zecchi** [reply]
Ok, thank you!
